src/partials/footer.handlebars

```diff
@@ -4,7 +4,7 @@
       <p>
         Copyright &copy; <span id="copyright-year">2015</span> <a href="http://apache.org">The Apache Software Foundation</a>,
         Licensed under the Apache License, Version 2.0<br>
-        Apache and the Apache feather logo are trademarks of The Apache Software Foundation.<br>
+        Apache and the Apache feather logo are trademarks of The Apache Software Foundation. | <a href="/privacy.html">Privacy Policy</a><br>
```

Comment:
@PramodSSImmaneni 

Both "/privacy.html" and "privacy.html" will work,  the former is an absolute path, and the later is a relative path. Once compiled all .html pages will locate in the root folder, so we can use either one. I see Apex site uses a lot of absolute paths, e.g. community.md (https://raw.githubusercontent.com/mx4web/apex-site/master/src/md/community.md).